db/migrate/20151123235600_add_purchase_type_to_gsa18f_procurements.rb

```diff
@@ -0,0 +1,11 @@
+class AddPurchaseTypeToGsa18fProcurements < ActiveRecord::Migration
+  def change
+    add_column :gsa18f_procurements, :purchase_type, :integer
+
+    execute <<-SQL
+      UPDATE gsa18f_procurements SET purchase_type = 0 WHERE purchase_type IS NULL;
+      SQL
```

Comment:
Seems that this is done within `change_column_null`, so don't think it's necessary.

http://apidock.com/rails/ActiveRecord/ConnectionAdapters/PostgreSQL/SchemaStatements/change_column_null

Ditto for the migration above.